SU-SOE/soe_basic
# D8CORE-3499: fixing the link icons

# READY FOR REVIEW

# Summary
- Removing code and making sure the external link gets the external icon link

# Review By (Date)
- 2/11

# Urgency
- Low

# Steps to Test

1.Pull in this change.
2. clear caches
3. look at a publication node page and see the internal link icons are the straight arrows and the external ones are the arrow pointing out.

# Affected Projects or Products
- SOE Publication theming

# Associated Issues and/or People
- D8CORE-3499

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
